resources/public/api/conf/1.0/common/errors.yaml

```diff
@@ -0,0 +1,208 @@
+components:
+  examples:
+
+    # 400
+
+    formatNino:
+      description: |
+        The format of the supplied NINO field is not valid.
+      value:
+        code: FORMAT_NINO
+        message: The provided NINO is invalid
+
+    formatTransactionId:
+      description: |
+        The format of the supplied transaction identifier field is not valid.
+      value:
+        code: FORMAT_TRANSACTION_ID
+        message: The provided transaction ID is invalid
+
+    formatFromDate:
+      description: |
+        The From date is not a valid ISO format date.
+      value:
+        code: FORMAT_FROM_DATE
+        message: The provided fromDate is invalid
+
+    formatToDate:
+      description: |
+        The To date is not a valid ISO format date.
+      value:
+        code: FORMAT_TO_DATE
+        message: The provided toDate is invalid
+
+    missingFromDate:
+      description: |
+        The From date parameter is missing.
+      value:
+        code: MISSING_FROM_DATE
+        message: The toDate has been provided, but fromDate is missing
+
+    missingToDate:
+      description: |
+        The To date parameter is missing.
+      value:
+        code: MISSING_TO_DATE
+        message: The fromDate has been provided, but toDate is missing
+
+    rangeToDateBeforeFromDate:
+      description: |
+        The To date is before the From date.
+      value:
+        code: RANGE_TO_DATE_BEFORE_FROM_DATE
+        message: The toDate cannot be earlier than the fromDate
+
+    ruleDateRangeInvalid:
+      description: |
+        The date range is longer than 732 days.
+      value:
+        code: RULE_DATE_RANGE_INVALID
+        message: The provided date range is invalid
+
+    ruleFromDateNotSupported:
+      description: |
+        The from date specified is before 2018-04-06.
+      value:
+        code: RULE_FROM_DATE_NOT_SUPPORTED
+        message:
```

Comment:
If an error lacks a message, I think it's best to just delete the message key entirely. Otherwise it comes up as "null" in the documentation: 
<img width="501" alt="image" src="https://user-images.githubusercontent.com/106095575/216003663-14ff8c68-8a04-44c1-a979-be06345853cd.png">